manifests/golang.yml

```diff
@@ -458,8 +458,6 @@ tests/:
       Test_Trace_Sampling_Resource: v1.60.0
       Test_Trace_Sampling_Tags: v1.60.0
       Test_Trace_Sampling_Tags_Feb2024_Revision: v1.64.0-dev
-    test_tracer.py:
-      Test_TracerSCITagging: v1.48.0
```

Comment:
we still need this so that if we run the tests against an older version they pass.